src/js/components/productSupplier/create/subsections/AdditionalDetails.jsx

```diff
@@ -106,20 +107,18 @@ const AdditionalDetails = ({ control, errors }) => {
 
 export default AdditionalDetails;
 
+export const additionalDetailsFormErrors = PropTypes.shape({
```

Comment:
Eslint doesn't shout at it? I remember there was a restriction about using `default export` and normal `export` in one file

Reply:
I had no such lint errors? Do we have this rule in our eslint config?